Survey/Configurations/Keychain/KeychainAccess.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  KeychainAccess.swift
+//  Survey
+//
+//  Created by Khanh on 02/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+import KeychainAccess
+
+private enum Keys: String {
+
+    case token = "token_info"
+}
+
+enum KeychainAccess {
+
+    private static let keychain = Keychain(service: Bundle.main.bundleIdentifier ?? "")
+
+    static var token: APIToken? {
```

Comment:
I think this should be KeychainToken but we can leave it for the storing authentication task. 
```suggestion
    static var token: KeychainToken? {
```